e2e/src/acceptanceTest/kotlin/maru/e2e/CliqueToPosTest.kt

```diff
@@ -159,13 +155,22 @@ class CliqueToPosTest {
     restartNodeFromScratch(nodeName, nodeEthereumClient)
     log.info("Container $nodeName restarted")
 
-    await
-      .pollInterval(1.seconds.toJavaDuration())
+    val awaitCondition =
+      if (nodeName.contains("nethermind")) {
+        await
+          .pollInterval(10.seconds.toJavaDuration())
```

Comment:
Is nethermind slower to startup?

Reply:
Yes. It sometimes doesn't peer on startup and that's why sometimes it has to be restarted here. And when it's being restarted, it takes much longer to start up than Erigon for example.

The first issue with peering would be better handled by restarts by Docker compose healthchecks instead, same as Geth or Erigon, but Nethermind's JsonRPC is protected with JWT, so it's not as easy to implement the healthcheck. I decided to do it this way than spending more time to address it in docker-compose at this stage.